src/main/scala/passera/README.md

```diff
@@ -0,0 +1,65 @@
+<!--
+  Copyright (c) 2011-2013, Nate Nystrom
```

Comment:
Copyright banners were added to all files that didn't have them. This makes it easier to reuse the source. It meets our source-code labeling requirements.